lib/features/family/features/scan_nfc/cubit/scan_nfc_cubit.dart

```diff
@@ -175,7 +175,7 @@ class ScanNfcCubit extends Cubit<ScanNfcState> {
         scanNFCStatus: ScanNFCStatus.error,
       ),
     );
-    NfcManager.instance.stopSession();
+    NfcManager.instance.stopSession(errorMessage: 'handleException');
```

Comment:
Is this an error message the user could see? (Same question for also all the others)

Reply:
Oops sorry, that was for debugging, will remove!